test/test.js

```diff
@@ -344,6 +344,13 @@ describe('Basic functionality', function() {
       .expectNoHeader('set-cookie')
       .expectNoHeader('set-cookie2', done);
   });
+
+  it('Mock a http status response if requested in the headers', function (done) {
+    request(cors_anywhere)
+      .get('/example.com/mockhttpstatus')
```

Comment:
The test doesn't work because the endpoint is not defined. To get the test to pass, a mock response must be defined here: https://github.com/Rob--W/cors-anywhere/blob/a0309e6c475036b4ba2a04372f8d5b6462bd4825/test/setup.js#L55